console/src/components/Services/Actions/Modify/ActionEditor.js

```diff
@@ -79,6 +80,7 @@ const ActionEditor = ({
   };
 
   const allowSave =
+    !readOnlyMode &&
```

Comment:
Unnecessary since we are hiding the buttons anyway.

Reply:
@wawhal Done